integration/tests/upgrades/protocol_version_upgrade_test.go

```diff
@@ -84,6 +76,7 @@ func (s *ProtocolVersionUpgradeSuite) TestProtocolStateVersionUpgradeServiceEven
 	// after an invalid protocol version upgrade event, we should still have a v0 kvstore
 	snapshot = s.AwaitSnapshotAtView(invalidUpgradeActiveView, time.Minute, 500*time.Millisecond)
 	actualProtocolVersion = snapshot.Encodable().SealingSegment.LatestProtocolStateEntry().KVStore.Version
+	// below line is failing, probably because we are starting with v0 now...
```

Comment:
English is not my first language, but to me the following would sound more natural:
```suggestion
	// line below is failing, probably because we are starting with v0 now...
```
Is line 79 still failing? And if so, why is the test not failing?

Reply:
No, just an outdated comment to myself while debugging - thanks for pointing it out.